src/topi/einsum.cc

```diff
@@ -98,24 +98,33 @@ EinsumEquation EinsumEquation::FromString(const std::string& equation) {
 }
 
 PrimExpr GetBroadcastedExtent(const PrimExpr& extent1, const PrimExpr& extent2) {
-  int64_t extent1_value = GetConstInt(extent1);
-  int64_t extent2_value = GetConstInt(extent2);
-  if (extent1_value == extent2_value) {
+  const IntImmNode* extent1_imm = extent1.as<IntImmNode>();
```

Comment:
One minor note for future, would be useful to have common broadcast extent handling in the future